src/BlazorWasmSdk/Sdk/Sdk.props

```diff
@@ -19,4 +19,12 @@ Copyright (c) .NET Foundation. All rights reserved.
   </PropertyGroup>
 
   <Import Project="$(_BlazorWebAssemblyPropsFile)" />
+
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports_BlazorWasm)' != 'true'
+      And '$(TargetFrameworkIdentifier)' == '.NETCoreApp'
+      And $([MSBuild]::VersionGreaterThanOrEquals('$(_TargetFrameworkVersionWithoutV)', '6.0'))">
+    <Import Include="Microsoft.Extensions.Configuration" />
+    <Import Include="Microsoft.Extensions.DependencyInjection" />
+    <Import Include="Microsoft.Extensions.Logging" />
```

Comment:
Ugh, is `Import` the itemgroup that we chose for this?

`Import` is already a keyword in MSBuild (see [here](https://docs.microsoft.com/en-us/visualstudio/msbuild/import-element-msbuild?view=vs-2019)), we should reconsider the name to avoid confusion.

This will just make navigating msbuild files more miserable

Reply:
Yea it's aligned with VB

Reply:
Yes, I brought up the same concern, but we felt aligning the behaviour is more important. Also, global using was partially inspired by the `Import` mechanism of VB it turns out.